internal/cmd/kafka/confluent_kafka.go

```diff
@@ -246,7 +246,7 @@ func consumeMessage(e *ckafka.Message, h *GroupHandler) error {
 	return nil
 }
 
-func runConsumer(consumer *ckafka.Consumer, groupHandler *GroupHandler) error {
+func RunConsumer(consumer *ckafka.Consumer, groupHandler *GroupHandler) error {
```

Comment:
All public functions shared across multiple packages should probably get moved to `internal/pkg/kafka` to avoid future import cycle issues.

Reply:
tried it and brought many import cycles from cmd, sr, and kafka. Will do that all in the clean-up pr as well.